src/Components/Header/Header.js

```diff
@@ -1,29 +1,32 @@
 import React from 'react';
 import logo from '../../Assets/logo.png';
-import {NavLink} from "react-router-dom"
+import { NavLink } from 'react-router-dom';
 import './Header.css';
 import LocalMallOutlinedIcon from '@mui/icons-material/LocalMallOutlined';
 import FavoriteBorderOutlinedIcon from '@mui/icons-material/FavoriteBorderOutlined';
-import { useCartContext } from '../../Contexts/CartContext';
-import { useAuthContext } from '../../Contexts/AuthContext';
-import { signOutUser } from '../../fireBaseAuth';
-import {Link} from "react-router-dom"
+import { useCartContext } from '../../export';
```

Comment:
The same comment as above, you can use barrel exports here to make the code looks cleaner.